quiz/system_tests/test_fixed_quiz.py

```diff
@@ -0,0 +1,31 @@
+from quiz import fixed_quiz
+from quiz.models import Quiz
+from quiz.system_tests.system_test_case import SystemTestCase
+from quiz.test_helpers import create_questions
+
+
+class FixedQuizzesTest(SystemTestCase):
+    def test_user_is_offered_to_start(self):
+        create_questions(fixed_quiz.nof_questions_in_quiz)
+        self.answer_random_questions()
+        self.should_be_offered_to_start_a_quiz()
+
+    def test_user_who_mistypes_a_quiz_url_gets_suggestions(self):
+        self.mistype_a_quiz()
+        self.should_see_suggestions_for_quizzes_with_similar_keys()
+
+    def answer_random_questions(self):
```

Comment:
Maybe rename to `view_random_question` or something? Since this function doesn't actually answer questions.